src/main/resources/processes/COMP/COMP_EXGRATIA_TRIAGE.bpmn

```diff
@@ -163,7 +163,7 @@
       <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${CctTriageAccept == "Yes"}</bpmn:conditionExpression>
     </bpmn:sequenceFlow>
     <bpmn:sequenceFlow id="SequenceFlow_0yakx66" sourceRef="ExclusiveGateway_1hbd7xu" targetRef="Screen_Transfer">
-      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${CctTriageAccept == "No" || CctTriageAccept == "CCH"}</bpmn:conditionExpression>
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${CctTriageAccept == "No"}</bpmn:conditionExpression>
```

Comment:
Should CCH be left in so we can eventually switch back to CCH once old cases are cleared out?